src/EditorFeatures/Core.Wpf/SymbolSearch/SymbolSearchUpdateEngine.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.CodeAnalysis.SymbolSearch
 {
     /// <summary>
     /// A service which enables searching for packages matching certain criteria.
-    /// It works against an <see cref="Microsoft.CodeAnalysis.Elfie"/> database to find results.
+    /// It works against a <see cref="Microsoft.CodeAnalysis.Elfie"/> database to find results.
```

Comment:
Curious on this case and others -- if this gets rendered as just "Elfie" in the final display of the comment, you'd want it 'an'....I'm not sure what is right though.

Reply:
Yeah.  this is tricky :)  I know i picked 'an' here because in my mind i was saying "an Elfie database".  But it really depends on final rendering.